src/common/home/Home.jsx

```diff
@@ -1,4 +1,4 @@
-import { useState, useEffect, useContext } from "react";
+import { useState, useEffect, useContext, useMemo,useRef, useCallback } from "react";
```

Comment:
@Anirban-1490 We are not using all the hooks. Please remove the ones from import that we are not using,

Reply:
ok understood